Source/WebGPU/WebGPU/WebGPU.h

```diff
@@ -1627,6 +1626,8 @@ WGPU_EXPORT void wgpuBufferRelease(WGPUBuffer buffer) WGPU_FUNCTION_ATTRIBUTE;
 #if ENABLE(WEBGPU_SWIFT)
 WGPU_EXPORT void wgpuBufferCopy(WGPUBuffer buffer, std::span<const uint8_t> data, size_t offset) WGPU_FUNCTION_ATTRIBUTE;
 #endif
+#else
+WGPU_EXPORT void * wgpuBufferGetBufferContents(WGPUBuffer buffer) WGPU_FUNCTION_ATTRIBUTE;
```

Comment:
does this function still exist (returning `void*`)? I don't see the implementation in the PR, but maybe it wasn't touched?

Reply:
It does not exist, I think I should just remove it instead of leaving the C definition